README.md

```diff
@@ -5,34 +5,34 @@
 * [Prerequisities](#prerequisites)
```

Comment:
@tlberglund since the heading changed, the TOC needs to be changed as well (lines 3-5)